src/App.tsx

```diff
@@ -28,6 +29,8 @@ function App() {
   const [isValidateSessionPending, setIsValidateSessionPending] = useState(false)
   const [isFinishValidateSessionPending, setIsFinishValidateSessionPending] = useState(false)
 
+  const [network, setNetwork] = useState(undefined as unknown as Chain)
```

Comment:
You can do this:

`const [network, setNetwork] = useState<undefined | Chain>()`